src/main/java/org/jboss/logmanager/formatters/Formatters.java

```diff
@@ -477,6 +483,30 @@ public void renderRaw(Formatter formatter, final StringBuilder builder, final Ex
         };
     }
 
+    private static DateTimeFormatter createDateTimeFormatter(String formatString) {
+        if ((formatString == null) || "ISO8601".equals(formatString)) { // Logback uses 'ISO8601'
```

Comment:
What happens if the name of a format could also be parsed as a valid date string? Should we perhaps use an indicator character to specify that the remainder of the string is actually an identifier instead of a pattern?

Reply:
I thought about that. I'm not sure what pattern would match a constant though. Since we will have control too, we could enforce they are not valid patterns.